### PR TITLE
[TASK] Add composer.json and template

### PR DESCRIPTION
**Pull request by kitsunet**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
+    "name": "neos/neos-development-collection",
+    "description": "Neos packages in a joined repository for pull requests.",
+    "type": "neos-package-collection",
+    "require": {
+        "typo3/flow": "*",
```
Comment (dfeyer) [inline composer.json]: Why not a specific version like the 2.0 branch composer.json ?

Comment (kitsunet) [inline composer.json]: This file is generated automatically. If it should be a version branch, then we would need to fix this in the package composer jsons inside.

Comment (aertmann) [inline composer.json]: should be ~2.3 if so..

Comment (kitsunet) [inline composer.json]: I would like to merge this as is for now just becaue that reflects the current state of this branch. We can then think about adjusting the inner composer manifests as we like and update this merged one.

Comment (skurfuerst) [inline composer.json]: agreed :D

**Comment by aertmann** [comment]
:+1:

**Pull request merged**